packages/search-adapter/src/types/response.types.ts

```diff
@@ -53,10 +53,10 @@ export interface TrackableShowResponse {
  * @public
  */
 export interface SearchResponse {
-  banners: Banner[];
+  banners?: Banner[];
   facets?: Facet[];
   partialResults: PartialResult[];
```

Comment:
Can we do the same with `partialResults`, `spellcheck` and `redirections`?

Reply:
Sure!